recipes-mender/mender/files/mender.service

```diff
@@ -5,7 +5,7 @@ Description=Mender OTA update service
 Type=simple
 User=root
 Group=root
-ExecStart=/usr/bin/mender -daemon
+ExecStart=/usr/bin/mender -daemon -log-level=debug -log-file=/var/log/mender.log
```

Comment:
"TODO: Remove this" type of tasks have a tendency to stick around longer than anticipated. It can easily get forgotten, and it's not given that everyone needs this (I would use a different approach in my work, for example). If you are debugging issues anyway, then why not enable it in your local build?